lib/src/index.node.ts

```diff
@@ -2,8 +2,8 @@ import { NanoTDFClient, NanoTDFDatasetClient, AuthProviders, version, clientType
 import fetch from 'node-fetch';
 import { webcrypto as crypto } from 'node:crypto';
 
-globalThis.crypto = crypto as unknown as Crypto;
-globalThis.fetch = fetch as typeof globalThis.fetch;
+globalThis.crypto ??= crypto as unknown as Crypto;
```

Comment:
no idea what `??=` does

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment

to avoid overwriting other library's poly fills